repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasAbstractDefStoreV2.java

```diff
@@ -51,6 +51,7 @@
 
     protected final AtlasTypeDefGraphStoreV2 typeDefStore;
     protected final AtlasTypeRegistry        typeRegistry;
+    public static AtlasTypeRegistry invalidKeywordsForTypeCreation = new AtlasTypeRegistry();
```

Comment:
```
protected final AtlasTypeRegistry typeRegistry;
```
Its already there, no point declaring another object of same class

Reply:
On creating new object of AtlasTypeRegistry, it has only primitive datatypes stored in RegistryData whereas this typeRegistry stores the name of the typesdef that I create.

RegistryData of new object of AtlasTypeRegistry:
<img width="143" alt="Screenshot 2023-02-15 at 6 26 50 PM" src="https://user-images.githubusercontent.com/121708100/219061861-77dbb2f1-a789-4e8d-b0bf-9b9d6f82dac3.png">

RegistryData of the existing typeRegistry:
<img width="270" alt="Screenshot 2023-02-15 at 8 20 32 PM" src="https://user-images.githubusercontent.com/121708100/219061774-9610fd96-92c7-400f-abdf-b21916d2a1d7.png">